src/certificates/certificates.module.ts

```diff
@@ -1,16 +1,17 @@
 import { Module } from '@nestjs/common';
-
 import {
   CertificateFilesController,
   CertificatesController,
 } from './controllers';
+import { TasksService } from './services';
 import { PrismaModule } from '../prisma/prisma.module';
 import { StorageModule } from '../storage/storage.module';
+import { ScheduleModule } from '@nestjs/schedule';
 import { CertificatesService, CertificateFilesService } from './services';
 
 @Module({
   controllers: [CertificatesController, CertificateFilesController],
-  providers: [CertificatesService, CertificateFilesService],
-  imports: [PrismaModule, StorageModule],
+  providers: [CertificatesService, CertificateFilesService, TasksService],
+  imports: [PrismaModule, StorageModule, ScheduleModule.forRoot()],
```

Comment:
@aarbanas Could you move the ScheduleModule into `app.module.ts`? If I remember correctly `.forRoot` registers the module on a global level so it's much nicer that it's in the "main" module as well.

Reply:
Nice catch. I've tested it and it works. Tnx